experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/Middleware/HandoffMiddleware.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.Bot.Builder;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+using LivePersonAgentBot.Bots;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Text;
```

Comment:
Remove and sort usings. There seems to be many not-necessary using statments, and their order seems random.